src/main/java/com/yahoo/identity/services/storage/sql/SqlStorage.java

```diff
@@ -32,8 +33,11 @@ public AccountCreate newAccountCreate() {
 
     @Nonnull
     @Override
-    public Account getAccount(@Nonnull String username) {
-        return new SqlAccount(sqlSessionFactory, username);
+    public AccountModel getAccount(@Nonnull String username) {
+        try (SqlSession session = sqlSessionFactory.openSession()) {
+            AccountMapper mapper = session.getMapper(AccountMapper.class);
```

Comment:
I wonder if we need to split the `getAccount` into two versions of mybatis mapping (one with private fields like email and one without)? So that `AnonymousSession` cannot touch the private entries in DB fundamentally.